klaus/markup.py

```diff
@@ -51,5 +51,20 @@ def render_rest(content):
     LANGUAGES.append((['.rst', '.rest'], render_rest))
 
 
-for loader in [_load_markdown, _load_restructured_text]:
+def _load_notebook():
+    try:
+        import nbformat
+        import nbconvert
+    except ImportError:
+        return
+
+    def render_notebook(content):
+        nb = nbformat.reads(content, nbformat.NO_CONVERT)
+        (output, resources) = nbconvert.HTMLExporter(template_file='./klaus/templates/my_full.tpl').from_notebook_node(nb)
```

Comment:
`my_full`? Confusing name :-D

Reply:
It's just a little CSS modification of [nbconvert template 'full'](https://github.com/jupyter/nbconvert/blob/d35c5155a597766b0ff74f930d0f39908f193f86/nbconvert/templates/html/full.tpl), cause the `width` in `@media` setting of that template will make the html overflow the markup `div`. The naming is not very clear, do you have any suggestions?

Reply:
I see! In this case could we simply use the default template and add the CSS fixes to klaus' CSS file? Or if that doesn't work, can we use inheritance to override the CSS instead of copy-pasting the whole file?